qemu/tests/driver_in_use.py

```diff
@@ -66,6 +67,12 @@ def run_bg_test_simu(bg_stress_test):
                 utils_test.run_virt_sub_test, (test, params, env),
                 {"sub_type": bg_stress_test})
             stress_thread.start()
+
+        if params.get("set_bg_stress_flag", "no") == "yes":
```

Comment:
should consider other bg test or set only balloon bg test.

Reply:
set_bg_stress_flag is only available for balloon cases

Reply:
I think it's better to add a comment or logging info to figure out "set_bg_stress_flag is only available for balloon cases" which will be make it more clear.

Reply:
Modified